src/index.html

```diff
@@ -11,7 +11,170 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset id="first">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            autocomplete="off"
+            name="surname"
+            required
+          >
+        </div>
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input
```

Comment:
accordingly to the task description, autocomplete should be off for all text inputs